### PR TITLE
Setup Travis CI Documentation

### PR DESCRIPTION
**Pull request by edi33416**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,28 @@
 sudo: false
 os:
- - linux
+    - linux
```
Comment (wilzbach) [inline .travis.yml]: Unrelated

Comment (edi33416) [reply .travis.yml]: Was auto-generated
